Entities.js

```diff
@@ -0,0 +1,339 @@
+import { StyleSheet, Text, View, Image } from 'react-native';
+import { Dimensions, Keyboard } from 'react-native';
+import { React, Component, useEffect, useState, useRef } from 'react';
+import { UseOnKeyPress, UseOnKeyRelease } from "./Key.js"
+import { CreateCircle, CreateBrick, DeleteEntity } from "./Dodge.js"
+import Matter from "matter-js"
+
+const MAXwidth = Dimensions.get('window').width;
+const MAXheight = Dimensions.get('window').height;
+let counter = 0
+let GlobalX = 400
+let GlobalY = 400
+let playerHeight = 150
+let playerWidth = 150
+let input = [0,0,0,0]
+//      left,right,up,down (w,a,s,d) each stores a value, 0 = not pressed
+
+//count intersections with players
+const Counter = () => {
+  console.log({counter})
```

Comment:
Probably move this to Dodge.js, change the logic to update it to be through this.parent.counter.